### PR TITLE
Build 2.0

### PR DESCRIPTION
**Pull request by R1ZZU**

### CODE REVIEW

**Comment by dzmitry-varabei** [comment]
Awesome :+1: 
but our favicon icon is missing in bin folder :sob:

**Comment by R1ZZU** [comment]
I assume `robots.txt` and `apple-touch-icon-precompressed.png` also should be copied to `dist` folder?

**Comment by dzmitry-varabei** [comment]
we can drop robots.txt but apple-touch-icon-precompressed.png can be useful

**Comment by R1ZZU** [comment]
Ok, it's already done in a8fe2b13ce426de32a72d3f8e8add5de71b166dc
@DaQuirm, can you please take a look? :alien:

**Comment by DaQuirm** [comment]
Taking a look, yessir :smiley_cat:

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+var del         = require('del');
+var minimist    = require('minimist');
+var runSequence = require('run-sequence');
+var pngquant    = require('imagemin-pngquant');
+var $           = require('gulp-load-plugins')();
+
+var options = {
+  src: 'app',
+  staging: '.tmp',
+  dest: 'appBin'
```
Comment (DaQuirm) [inline gulpfile.js]: Why do we use camel-case here?..

**Comment by DaQuirm** [comment]
The mustache task overwrites archive.html in /app. Should it do that?

File: gulpfile.js (hunk 1 above)
Comment (R1ZZU) [inline gulpfile.js]: Only god knows :smile_cat:

**Comment by R1ZZU** [comment]
We can drop it at all, and write rendered mustache template to `.tmp` folder

File: gulpfile.js (hunk 1 above)
Comment (DaQuirm) [inline gulpfile.js]: I vote for ditching camelCase and adopt snake case project-wide

**Comment by R1ZZU** [comment]
I think we can create a couple of issues related to `camelCase` :arrow_right:  `snake-case` and mustache task. Any other comments?

**Comment by DaQuirm** [comment]
Not from me, no. LGTM :smiley_cat:

**Comment by R1ZZU** [comment]
#16 
#17 
Created :smiley:

**Comment by DaQuirm** [comment]
You're a superhero! :clap:

**Comment by R1ZZU** [comment]
ROFL :laughing:

**Comment by DaQuirm** [comment]
Come on, let's merge it and have an awesome build process! :shipit:

**Pull request merged**